corehq/apps/users/views/mobile/users.py

```diff
@@ -136,11 +147,12 @@ def custom_data(self):
     def editable_user(self):
         try:
             user = CommCareUser.get_by_user_id(self.editable_user_id, self.domain)
-            if not user:
-                raise Http404()
-            return user
         except (ResourceNotFound, CouchUser.AccountTypeError, KeyError):
             raise Http404()
+        else:
```

Comment:
good point.  Also, on further inspection, I'm having a hard time figuring out how user could ever be falsy.